api/src/main/clojure/xtdb/api.clj

```diff
@@ -162,22 +162,25 @@
    (-> @(q& node q+args opts)
        (rethrowing-cause))))
 
+(defn ->QueryOptions [{:keys [args after-tx basis tx-timeout default-tz default-all-valid-time? explain? key-fn], :or {key-fn :kebab-case-keyword}}]
```

Comment:
this is in the `xtdb.api` namespace - can you confirm you want it to become part of the public API?